docs/resources/cluster_v2.md

```diff
@@ -236,6 +236,96 @@ The `<AUTH_CONFIG_SECRET_NAME>` represents a generic kubernetes secret which con
 
 Many registries may be specified in the `rke_config`s `registries` section, however the `system-default-registry` from which core system images are pulled is always denoted via the `system-default-registry` key of the `machine_selector_config` or the `machine_global_config`. For more information on private registries, please refer to [the Rancher documentation](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/global-default-private-registry#setting-a-private-registry-with-credentials-when-deploying-a-cluster) 
 
+### Creating Rancher V2 cluster with cluster agent customization. For Rancher v2.7.5 and above.
+
+```hcl
+resource "rancher2_cluster_v2" "foo" {
+  name = "foo"
+  enable_network_policy = false
+  rke_config {
+    machine_pools {
+      name = "pool1"
+      cloud_credential_secret_name = rancher2_cloud_credential.foo.id
+      control_plane_role = true
+      etcd_role = true
+      worker_role = true
+      quantity = 1
+      machine_config {
+        kind = rancher2_machine_config_v2.foo.kind
+        name = rancher2_machine_config_v2.foo.name
+      }
+    }
+  }
+  cluster_agent_deployment_customization {
+    append_tolerations {
+      effect = "NoSchedule"
+      key    = "tolerate/control-plane"
+      value  = "true"
+}
+    override_affinity = <<EOF
+{
+  "nodeAffinity": {
+    "requiredDuringSchedulingIgnoredDuringExecution": {
+      "nodeSelectorTerms": [{
+        "matchExpressions": [{
+          "key": "not.this/nodepool",
+          "operator": "In",
+          "values": [
+            "true"
+          ]
+        }]
+      }]
+    }
+  }
+}
+EOF
+    override_resource_requirements {
+      cpu_limit      = "800"
+      cpu_request    = "500"
+      memory_limit   = "800"
+      memory_request = "500"
+    }
+  }
+}
+```
+
+### Creating Rancher V2 cluster with Pod Security Policy Admission Configuration Template (PSACT). For Rancher v2.7.2 and above.
+
+**Note** When PSACT is enabled in RKE2 or K3s, Rancher webhook sets the `kube-apiserver-arg` to the Pod Security Admission mount path. To suppress Terraform constantly trying to reconcile that arg, it must be set locally.
```

Comment:
small nit, Note should be followed by a colon -> `**Note:**`. I know you weren't the one who wrote this initially, but if we are fixing up wording and whatnot it couldn't hurt to add it in. lmk if you disagree, again its a small nit